src/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -188,6 +188,7 @@ const CreateScenarioDialog = ({
 
   return (
     <Dialog
+      data-cy="scenario-create-dialog"
```

Comment:
could you please rename it to `create-scenario-dialog` to match with component name?